homeassistant/components/fritz/device_tracker.py

```diff
@@ -14,13 +14,14 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-CONF_DEFAULT_IP = "169.254.1.1"  # This IP is valid for all FRITZ!Box routers.
+DEFAULT_HOST = "169.254.1.1"  # This IP is valid for all FRITZ!Box routers.
+DEFAULT_USERNAME = "admin"
```

Comment:
This may be considered a breaking change

Reply:
I am not sure what you are referring to here. What could break here?

Reply:
The default username was previously `""`. I am not familiar with this platform, but if that is a valid username for this device, then a user who did not set that config value previously would have an unexpected username mismatch.